101-webapp-linux-sonarqube-azuresql/azuredeploy.json

```diff
@@ -0,0 +1,258 @@
+            "type": "string",
+            "defaultValue": "GP_S_Gen5_2",
+            "allowedValues": [
+                "GP_Gen4_1",
+                "GP_Gen4_2",
+                "GP_Gen4_3",
+                "GP_Gen4_4",
+                "GP_Gen4_5",
+                "GP_Gen4_6",
+                "GP_Gen4_7",
+                "GP_Gen4_8",
+                "GP_Gen4_9",
+                "GP_Gen4_10",
+                "GP_Gen4_16",
+                "GP_Gen4_24",
+                "GP_Gen5_2",
+                "GP_Gen5_4",
+                "GP_Gen5_6",
+                "GP_Gen5_8",
+                "GP_Gen5_10",
+                "GP_Gen5_12",
+                "GP_Gen5_14",
+                "GP_Gen5_16",
+                "GP_Gen5_18",
+                "GP_Gen5_20",
+                "GP_Gen5_24",
+                "GP_Gen5_32",
+                "GP_Gen5_40",
+                "GP_Gen5_80",
+                "GP_S_Gen5_1",
+                "GP_S_Gen5_2",
+                "GP_S_Gen5_4"
+            ],
+            "metadata": {
+                "description": "Azure SQL Database SKU Name"
+            }
+        },
+        "sqlDatabaseSkuSizeGB": {
+            "type": "int",
+            "minValue": 1,
+            "maxValue": 1024,
+            "defaultValue": 16,
+            "metadata": {
+                "description": "Azure SQL Database Storage Max Size in GB"
+            }
+        },
+        "location": {
+            "type": "string",
+            "defaultValue": "[resourceGroup().location]",
+            "metadata": {
+                "description": "Location for all the resources."
+            }
+        }
+    },
+    "variables": {
+        "databaseName": "sonarqube",
+        "sqlServerName": "[concat(parameters('siteName'), '-sql')]",
+        "servicePlanName": "[concat(parameters('siteName'), '-asp')]",
+        "servicePlanPricingTiers": {
+            "F1": {
+                "tier": "Free"
+            },
+            "B1": {
+                "tier": "Basic"
+            },
+            "B2": {
+                "tier": "Basic"
+            },
+            "B3": {
+                "tier": "Basic"
+            },
+            "S1": {
+                "tier": "Standard"
+            },
+            "S2": {
+                "tier": "Standard"
+            },
+            "S3": {
+                "tier": "Standard"
+            },
+            "P1V2": {
+                "tier": "Standard"
+            },
+            "P2V2": {
+                "tier": "Standard"
+            },
+            "P2V3": {
+                "tier": "Standard"
+            }
+        },
+        "sqlDatabaseServerlessTiers": [
+            "GP_S_Gen5_1",
+            "GP_S_Gen5_2",
+            "GP_S_Gen5_4"
+        ]
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Web/serverfarms",
+            "apiVersion": "2018-02-01",
+            "name": "[variables('servicePlanName')]",
+            "location": "[resourceGroup().location]",
```

Comment:
all location properties must be [parameters('location')]

Reply:
Done.